test/e2e/sourcery.js

```diff
@@ -42,8 +43,23 @@ module.exports = {
             }
         });
         //Assert if expected user is logged in
-        browser.assert.containsText('.user-info__title', 'Demo User')
-            .saveScreenshot(conf.imgpath(browser) + 'Demo.png')
-            .end();
+        browser.assert.containsText('.user-info__title', 'Deividas Vaškevicius')
+            .saveScreenshot(conf.imgpath(browser) + 'Demo.png');
+
+        browser.assert.containsText('.main-nav', 'Time Logging');
+        browser.assert.containsText('.main-nav', 'Invoices');
+        browser.assert.containsText('.main-nav', 'Projects');
+        browser.assert.containsText('.main-nav', 'Clients');
+        browser.assert.containsText('.main-nav', 'Time Entries');
+        
+        browser.assert.containsText('.main-nav__link.main-nav__link--active', 'Time Logging');
```

Comment:
Could this be shortened to '.main-nav__link--active'?

Reply:
yes, it could